src/HttpExponentialBackoff.js

```diff
@@ -127,6 +127,25 @@ class HttpExponentialBackoff {
       return timeToWait
     }
   }
+
+  /**
+   * Retry Delay returns a function that either:
+   *  - return the value of Retry-After header, if present
+   *  - implements exponential backoff otherwise
+   *
+   * @param {number} initialDelayInMillis The multiplying factor and the initial delay. Eg. 100 would mean the retries would be spaced at 100, 200, 400, .. ms
+   * @returns {Function} retryDelayFunction {function(*=, *, *): number}
+   * @private
+   */
+  __getRetryDelayWithRetryAfterHeader (initialDelayInMillis) {
+    return (attempt, error, response) => {
+      const retryAfter = response.headers.get('Retry-After')
```

Comment:
we need to ensure it is case-insensitive.

Reply:
I thought it was already as it is part of a fetch response, I was looking here: https://developer.mozilla.org/en-US/docs/Web/API/Headers/get